app/views/metadata_presenter/page/form.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="fb-main-grid-wrapper" data-block-id="<%= @page.id %>" data-block-type="page" data-block-pagetype="<%= @page.type %>">
+<div class="fb-main-grid-wrapper" data-fb-pagetype="<%= @page.type %>" data-here="true">
```

Comment:
`data-here`?

Reply:
Haha! Oops! That was me wondering why I wasn't seeing any changes (Gemfile issues). Will remove.